tests/ui/hidden_static_lifetime.stderr

```diff
@@ -0,0 +1,16 @@
+error: this lifetime can be changed to `'static`
+  --> $DIR/hidden_static_lifetime.rs:17:6
+   |
+LL | fn a<'a>() -> &'a str {
+   |      ^^ help: try: `changing the lifetime `'a` to 'static`
```

Comment:
More cents: Avoid "help: try:". The fix seems incorrect. Shouldn't the correct help be "Try removing the lifetime-parameter and change references to it to `'static`"

Reply:
Suggestion applied.